MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/model/ExampleItemModel.java

```diff
@@ -60,6 +65,16 @@ public ExampleItemModel(int title, int description, Intent activity, int imageUr
     this.showNewIcon = showNewIcon;
   }
 
+  public ExampleItemModel(int title, int description, Intent activity, int imageUrl,
+                          boolean showNewIcon, int minSdkVersion) {
+    this.title = title;
+    this.description = description;
+    this.imageUrl = imageUrl;
+    this.activity = activity;
+    this.showNewIcon = showNewIcon;
+    this.minSdkVersion = minSdkVersion;
+  }
+
   public ExampleItemModel(int title, int description, Intent activity, int imageUrl) {
```

Comment:
Is the previous constructor necessary to keep?

Reply:
Removed